docs/usage.md

```diff
@@ -95,6 +119,30 @@ genome: 'GRCh37'
 
 You can also generate such `YAML`/`JSON` files via [nf-core/launch](https://nf-co.re/launch).
 
+### Decontamination
+
+Filter the output files from metagenomics classifiers like `Kraken2`, `Centrifuge`, or `DIAMOND` to remove false positives and background contamination can be activated by enabling `--decontamination` option. This step compares results to the negative control to identify likely present species based on user-defined thresholds.
+
+### Extract Viral TaxIDs
+
+This step involves extracting all taxonomic IDs of viral species predicted by classifiers by enabling `--perform_extract_reads`, while not specifying a taxonomic ID list with `--taxid`.
+
+### Extract Reads
+
+This step either retrieves the reads of all viral TaxIDs predicted by classifiers or extracts reads from a user-defined list of TaxIDs separated by spaces when the `--taxid` option is activated. Extracting reads predicted by `Kraken2` can be activated with `--extract_kraken2_reads`, extracting reads predicted by `Centrifuge` can be activated with `--extract_centrifuge_reads` and extracting reads predicted by `DIAMOND` can be activated with `--extract_diamond_reads`.
```

Comment:
So, this step does both?

Reply:
Yes, it either extracts reads for identified viral taxids or for a user-defined list of taxids